src/utils/canvas.ts

```diff
@@ -336,3 +336,23 @@ export async function renderChartImage({
     },
   })
 }
+
+export function handleTextOverflow(
+  c: CanvasRenderingContext2D,
+  text: string,
+  maxWidth: number
+): string {
+  let width = c.measureText(text).width
+  const ellipsis = "…"
+  const ellipsisWidth = c.measureText(ellipsis).width
+  if (width <= maxWidth || width <= ellipsisWidth) {
+    return text
+  } else {
```

Comment:
redundant else

Reply:
i dont really understand this comment, can u recommend me a way to fix this